bessctl/conf/testing/module_tests/random_drop.py

```diff
@@ -0,0 +1,47 @@
+import sugar
+from module import *
+from port import *
+
+## CRASH TESTS ##
+rd0 = RandomDrop(drop_rate=0)
+CRASH_TEST_INPUTS.append([rd0, 1, 1])
+
+## OUTPUT TESTS ##
+rd1 = RandomDrop(drop_rate=0)
+test_packet = gen_packet(scapy.TCP, '22.22.22.22', '22.22.22.22')
+OUTPUT_TEST_INPUTS.append([rd1, 1, 1,
+                           [{'input_port': 0,
+                               'input_packet': test_packet,
+                                'output_port': 0,
+                                'output_packet': test_packet}]])
+
+## CUSTOM TESTS ##
+def half_drop_test():
```

Comment:
One more tiny edit -- can you rewrite this test to test more than drop rate = 0.5 (like test at 0.75, 0.9, 0.3 as well?). You should be able to use function currying / partial evaluation to achieve this pretty quickly with your existing code (https://www.pydanny.com/python-partials-are-fun.html).